codec-http/src/main/java/io/netty/handler/codec/http/HttpUtil.java

```diff
@@ -42,4 +47,252 @@ public static boolean isAsteriskForm(URI uri) {
                 uri.getHost() == null && uri.getAuthority() == null && uri.getQuery() == null &&
                 uri.getFragment() == null;
     }
+
+    /**
+     * Returns {@code true} if and only if the connection can remain open and
+     * thus 'kept alive'.  This methods respects the value of the
```

Comment:
Will submit followup commit to fix these comments @normanmaurer. All of this was just copy/paste.